examples/gcal_oo_or.ty

```diff
@@ -90,7 +91,16 @@ p.add(
         Learning rate for the lateral excitatory projection to V1."""),
 
     inh_lr=param.Number(default=0.3,bounds=(0.0,None),doc="""
-        Learning rate for the lateral inhibitory projection to V1."""))
+        Learning rate for the lateral inhibitory projection to V1."""),
+
+    gpu=param.Boolean(default=False,bounds=(0,1),doc="""
+        Sets whether the simulation should be run on a GPU"""))
+
+
+if p.gpu:
+    from topo.gpu.projection import GPUSparseCFProjection
+    from topo.gpu.sheet import compute_sparse_gpu_joint_norm_totals, GPUSettlingCFSheet
+
```

Comment:
It's certainly possible to change runtests.py to allow an argument, but that's some horrific code, so maybe not worth messing with. Instead, you can just add the "gpu" option to examples/gcal_sparse.ty, but leave it turned off by default, then add a new file in topo/tests that does nothing but call examples/gcal_sparse.ty with gpu=True. That way there's only one file in examples, and only a few characters in the extra file in topo/tests. 



-- 
The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.